README.md

```diff
@@ -1,40 +1,54 @@
 
-# Eirini - what this?
+# What is Eirini?
 
-Eirini is a Kubernetes backend for Cloud Foundry. It deploys CF apps to a kube backend, using OCI images and Kube deployments.
+*Eirini* is a Kubernetes backend for Cloud Foundry.
+It deploys CF apps to a kube backend, using OCI images and Kube deployments.
 
-_But there's more!_
+Eirini gives you the nice integrated `cf push` flow,
+with CF Apps mapped directly to kube `StatefulSet`.
+In other words it decouples buildpack staging and stateless-multitenant-app running.
 
-Eirini exports staged CF images as docker images. So you can schedule them however you'd like. *And separately* it gives you the nice integrated `cf push` flow, with CF Apps mapped directly to kube Deployment objects. In other words it decouples buildpack staging and stateless-multitenant-app running.
+Since scheduling is increasingly commoditized,
+Eirini provides an "Orchestrator Platform Interface (OPI)" layer,
+that abstracts away orchestration from Cloud Foundry's control plane.
+This means Eirini is not solely a Kube backend at all,
+but that it is a generic backend for any scheduler!
+This means it could schedule to diego, kube, swarm and other orchestration platforms,
+as long as there is an implementation of the OPI layer for the target platform.
 
-_But there's more!_
+To offer a generic orchestrator interface,
+Eirini uses the diego abstractions of LRPs and Tasks to capture Cloud Foundry's notion of long running processes and one-off tasks.
 
-Eirini uses a little abstraction library, "OPI", which means it's not actually a Kube backend at all: it's a generic backend for any scheduler! This means it can schedule to diego/kube/swarm and whatever else is cool next year.
+Deployment instructions are available at: [cloudfoundry-incubator/eirini-release](https://github.com/cloudfoundry-incubator/eirini-release)
 
-It uses the diego abstractions -- LRPs and Tasks -- in order to support generic orchestrators.
-
-Deployment instructions are available at https://github.com/cloudfoundry-incubator/eirini-release
-
-# What components?
+# Eirini Components
 
 Eirini has the following components, the first two are available as subcommands of the `eirini` binary:
  
  - `Bifrost` converts and transfers cloud controller app specific requests to OPI specific objects and runs them in Kubernetes. It relies on the [`bits-service`](https://github.com/cloudfoundry-incubator/bits-service) to serve OCI images for droplets, and `OPI` to abstract the communication with Kube.
- - `OPI` or the "orchestrator provider interface" provides a declarative abstraction over multiple schedulers inspired by Diego's LRP/Task model and Bosh's CPI concept.
+ - `OPI` or the "Orchestrator Provider Interface" provides a declarative abstraction over multiple schedulers inspired by Diego's LRP/Task model and Bosh's CPI concept.
  - `Stager` implements staging by running Kubernetes/OPI one-off tasks
  
-# Tell me more 'bout OPI
+# Orchestrator Platform Interface (OPI)
```

Comment:
Just recognised this - I believe we call it Orchestrator ~Platform~**Provider** Interface.